pkg/api/v1alpha1/usersignup_types.go

```diff
@@ -3,6 +3,8 @@ package v1alpha1
 type SignupStatusReason = string
 
 var SignedUp SignupStatusReason = "SignedUp"
+var NotSignedUp SignupStatusReason = "NotSignedUp"
+var UnknownSignUpStatus SignupStatusReason = "Unknown"
```

Comment:
If I'm not mistaken we could use `const` here


```suggestion
const NotSignedUp SignupStatusReason = "NotSignedUp"
const UnknownSignUpStatus SignupStatusReason = "Unknown"
```